### ISSUE TITLE
Document accommodation plans as part of the Service Design Blueprint

### ISSUE THREAD

**Issue by mgifford** [GSA/openacr#247]
If an agency needs a digital product or service to be accessible to meet the accommodation needs.

The mitigation of Section 508 failures can often be addressed through custom JavaScript or workarounds. This needs to be fit within the Service Design Blueprint.
